### PR TITLE
[2021.2][URP] Update URP Lit SubTarget to generate emission color property

### PR DESCRIPTION
**Pull request by jessebarker**
### Checklist for PR maker
- [x] Have you added a backport label (if needed)? For example, the `need-backport-*` label. After you backport the PR, the label changes to `backported-*`.
- [x] Have you updated the changelog? Each package has a `CHANGELOG.md` file.

---
### Purpose of this PR
This PR fixes https://fogbugz.unity3d.com/f/cases/1319637/
It should also fix https://fogbugz.unity3d.com/f/cases/1324684/

The Universal Lit Shader Graph SubTarget was not adding the _EmissionColor property to its generated shaders, which caused the registered material editor to generate errors when a material was created with the Lit Shader Graph.  This PR simply updates the Lit SubTarget to add that property.

It might be good in the future if the PBRMasterGUI lived with the Universal Target code, rather than in Shader Graph, so these pieces could be more tightly coupled.

---
### Testing status
Manual testing using the repro steps in the bug.
Yamato results to come

---
### Comments to reviewers
I would like to confirm that the default value for this property is correct (easily changed)

### CODE REVIEW

File: com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalLitSubTarget.cs [hunk 1]
```diff
@@ -128,6 +129,20 @@ public override void GetActiveBlocks(ref TargetActiveBlockContext context)
...
+        {
+            base.CollectShaderProperties(collector, generationMode);
+            collector.AddShaderProperty(new ColorShaderProperty()
+            {
+                overrideReferenceName = "_EmissionColor",
+                hidden = true,
+                overrideHLSLDeclaration = true,
+                hlslDeclarationOverride = HLSLDeclaration.UnityPerMaterial,
+                value = new Color(1.0f, 1.0f, 1.0f, 1.0f)
+            });
```
Comment (cdxntchou) [inline com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalLitSubTarget.cs:142]: What does the _EmissionColor property do?

Comment (cdxntchou) [reply com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalLitSubTarget.cs:142]: Aside from getting rid of the warning spew?  :)

Comment (jessebarker) [reply com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalLitSubTarget.cs:142]: No idea.  It's not really in Shader Graph.  It's really a pipeline thing (even though the material editor GUI is in SG).

Comment (jessebarker) [reply com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalLitSubTarget.cs:143]: So, there is a custom ShaderGUI registered by the URP Lit SubTarget which "asserts" this property on the material editor.  If it's not there, we get the error.  It's always declared by the hand-written Lit.shader.  This adds it in the same fashion. The difference is the default value, which is the one thing I'm still not sure about (Lit.shader has it as a 3-component 0).

Comment (cdxntchou) [reply com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalLitSubTarget.cs:143]: Right, but is it the correct thing to add this property, or remove the warning?   Need to find someone who can answer that question..  :)

Comment (cdxntchou) [reply com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalLitSubTarget.cs:143]: ah, nevermind, I read the bug -- Kasper says to do it, he should know..   :)

Comment (cdxntchou) [reply com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalLitSubTarget.cs:143]: he should know the correct default as well.

Comment (cdxntchou) [reply com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalLitSubTarget.cs:143]: @KEngelstoft mind enlightening us as to what this value does?   :)

Comment (jessebarker) [reply com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalLitSubTarget.cs:143]: Yeah, I'd put him on this PR as reviewer for precisely that reason :-)

Comment (cdxntchou) [reply com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalLitSubTarget.cs:143]: ok, so it's kind of equivalent to MainTex/MainColor -- a fixed function definition of a GI input.
MainTex/MainColor define the bounce color for GI,
and _EmissionMap/_EmissionColor define the emission color for GI.

Comment (KEngelstoft) [reply com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalLitSubTarget.cs:143]: In the built in Standard shader, there is a checkbox for Emission. Once checked, the emission HDR color property is shown and the default color is black.
There is an enum dropdown letting the user choose if emission should control realtime, baked or no GI.
image

**Review by KEngelstoft — COMMENTED** [review]
Please change default color to black.

**Pull request closed without merging**